packages/wrangler/src/user.tsx

```diff
@@ -1155,10 +1170,8 @@ export function ChooseAccount(props: {
 /**
  * Ensure that a user is logged in, and a valid account_id is available.
  */
-export async function requireAuth(
-  config: Config,
-  isInteractive = true
-): Promise<string> {
+export async function requireAuth(config: Config): Promise<string> {
+  const isInteractive = process.stdin.isTTY;
```

Comment:
Trying to use `process.stdout.isTTY` would cause tests to throw errors from Ink 
```
ERROR  Raw mode is not supported on the current process.stdin, which Ink uses as input stream by default.
        Read about how to prevent this error on https://github.com/vadimdemedes/ink/#israwmodesupported
```